Gopkg.lock

```diff
@@ -495,8 +490,8 @@
     "width",
   ]
   pruneopts = "NUT"
-  revision = "342b2e1fbaa52c93f31447ad2c6abc048c63e475"
-  version = "v0.3.2"
+  revision = "f21a4dfb5e38f5895301dc265a8def02365cc3d0"
+  version = "v0.3.0"
```

Comment:
Something is broken there. Please restore `Gopkg.toml` / `Gopkg.lock` from `master` branch, then run `dep ensure -v` (without `-update`).